stock-usecase/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/DropdownComponents.kt

```diff
@@ -473,17 +476,18 @@ fun openOrgUnitTreeSelector(
         .showAsDialog()
         .singleSelection()
         .orgUnitScope(OrgUnitSelectorScope.ProgramCaptureScope(settingsUiState.programUid))
-        .withPreselectedOrgUnits(orgUnitData?.let { listOf(it.uid()) } ?: emptyList())
+        .withPreselectedOrgUnits(
+            viewModel.settingsUiState.value.facility?.let { listOf(it.uid()) } ?: emptyList()
```

Comment:
you already have the settingsUiState, you can do: `settingsUiState.facility?let{....}`